src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/config/RedirectConfigurationsBuilder.kt

```diff
@@ -0,0 +1,51 @@
+package it.pagopa.ecommerce.eventdispatcher.config
+
+import it.pagopa.ecommerce.commons.exceptions.RedirectConfigurationException
+import it.pagopa.ecommerce.commons.exceptions.RedirectConfigurationType
+import java.net.URI
+import java.util.function.Predicate
+import java.util.stream.Collectors
+import lombok.extern.slf4j.Slf4j
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * Configuration class used to read all the PSP configurations that will be used during redirect
+ * transaction
+ */
+@Configuration
+@Slf4j
+class RedirectConfigurationsBuilder {
+  /**
+   * Create a Map &lt String,URI &gt that will associate, to every handled PSP, the backend URI to
+   * be used to perform Redirect payment flow api call
+   *
+   * @param paymentTypeCodeList
+   * - set of all redirect payment type codes to be handled flow
+   * @param pspUrlMapping
+   * - configuration parameter that contains PSP to URI mapping
+   * @return a configuration map for every PSPs
+   */
+  @Bean
+  fun redirectBeApiCallUriMap(
+    @Value("\${redirect.paymentTypeCodeList}") paymentTypeCodeList: Set<String>,
```

Comment:
Not a list :)

```suggestion
    @Value("\${redirect.paymentTypeCodes}") paymentTypeCodes: Set<String>,
```

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/237/commits/357aea018bb253c357a9a0930336a91d41483101